services/core-web/src/components/mine/AdministrativeAmendment/MineAdministrativeAmendmentTable.js

```diff
@@ -156,6 +134,7 @@ export class MineAdministrativeAmendmentTable extends Component {
         <div className="cap-col-height">
           {(trigger &&
             trigger.length > 0 &&
+            // eslint-disable-next-line react/jsx-key
```

Comment:
I maybe got a bit lazy here- I just have no idea how "item" is structured and what's appropriate to use as a key.